language/evm/move-to-yul/tests/test-dispatcher/DispatcherEncodingStorage.move

```diff
@@ -0,0 +1,33 @@
+#[contract]
+module 0x2::M {
+    use Std::Vector;
+    use Evm::U256::{Self, U256};
+    use Evm::Evm::sign;
+
+    struct T<Elem: drop> has drop, key { v: vector<Elem> }
+
+    struct State has drop, key {
+        s1: vector<u8>,
+        s2: vector<U256>,
+    }
+
```

Comment:
I noticed we don't have any #evm_test for this. Is it because of the issue we talked about (the issue where you have something like `mload($locals)` on the left hand side)? If so, we can land this and reiterate.

Reply:
It is just because #evm_test cannot test the dispatcher along the way. We could go back revisit all tests for dispatcher later.